Userland/js.cpp

```diff
@@ -159,19 +159,96 @@ static void print(JS::Value value)
     putchar('\n');
 }
 
+String sanitize_repl_inputs(String input)
+{
+    StringBuilder result;
+    for (char c : input) {
+        if (c != ' ' && c != '\n') {
+            result.append(c);
+        }
+    }
+    return result.to_string();
+}
+
+void handle_repl_commands(String part, JS::Interpreter& interpreter)
+{
+    enum ReplCommands {
```

Comment:
Let's call this `ReplCommand` since it represents a single command. Also, let's make it an `enum class`.